BENJAMIN JUNG FAIR
============

---

> <bfair.kun@gmail.com> • (248) 318-7163

----

Relevant technical experience/skills
--------------------

**Computational biology**


- **General computation skills:** Linux, version control (git), high-performance computing cluster environments, reproducible virtual environments, notebooking/communication of analyses
- **Scripting languages:** python, R/tidyverse, workflow language (snakemake), shell scripting
- **NextGenSequencing (NGS) analysis:** Differential expression and splicing analysis, variant calling, motif analysis, splicing prediction tools
- **Public databases:** Working knowledge and experience using genetics/NGS databases: TCGA, GTEx, GWAS Catalog, 1000GenomesProject, ClinVar, OMIM, SRA
- Comfortable implementing third-party packages and command-line software for analysis
- Particular expertise in RNA splicing and quantitative genetics


**Wet lab**


- NGS library preparation: whole genome sequencing, RNA-seq, Chromatin-profiling (Cut&Tag assay)
- Operation of automated pipetting robot for NGS library prep, qPCR, etc
- General knowledge of lab techniques including human cell culture
- Comfortable with troubleshooting, and optimization, and methods development processes
- Purification of commonly used molecular biology enzymes on large scale (*Taq* polymerase, reverse transcriptase, tn5)
- Rudimentary knowledge of techniques for ipsc differentiation, embryoid-body and organoid culture, single-cell transcriptomics.
	- Most of my knowledge on these topics is from being in the lab environment where most of my colleuges work on these topics, rather than direct experience on these topics

Education
---------

2012-2018
:   **PhD, Biochemistry, Cell, and Molecular Biology**; Cornell University (Ithaca, NY, USA)

	**Minor: Bioinformatics**

______

2007-2010
:   **BA, Biology (*Cum laude*)**; Grand Valley State University (Allendale, MI, USA)

    **Minor: Chemistry**

Research Experience
---
2018-Present
:	**Postdoctoral scholar**, University of Chicago, Chicago, IL

	**Avisor**: Yang Li and Yoav Gilad (co-advised)

	**Main projects**:

	- Exploring molecular mechanisms and potential clinically-relevant gene targets of novel splicing-modifying small molecules - utilizing primarily transcriptomic analyses after *in vitro* treatment of known and novel small molecules.
	- Quantifying the relative contributions of various general gene expression mechanistic models (eg enhancer-promoter interactions, RNA-splicing, RNA decay) towards heritable complex traits. I am utilizing novel and published populaion scale -omics datasets (RNA-seq, ChIP-seq, >1000 datasets across 10 NGS assays in total) to comprehensively map genotype/phenotype associations for molecular traits (eg gene expression, splicing, enhancer activity) and systematically integrating these genotype/phenotype maps with GWAS loci.
	- Used comparative transcriptomics to identify genetic and non-genetic sources of variability in gene expression across chimpanzee and human populations.

2012-2018
:	**Graduate research assistant**, Cornell University, Ithaca, NY

	**Avisor**: Jeffrey A Pleiss

	**Thesis title**: Splicing pathway mutations identified in fission yeast via targeted RNA-seq based screens

	- Co-invented targeted RNA-sequencing NGS methodology (Multiplexed-primer extension sequencing)
	- Developed an RNA-seq based strategy to measure RNA splicing in thousands of mutant yeast strains in parallel
	- Identified mutations in chromatin remodeling enzymes and 3' end processing genes which impact splicing. Identified novel temperature-sensitive alleles of spliceosomal genes

2011-2012
:	**Undergraduate researcher**, Grand Valley State University, Allendale, MI

	**Avisor**: Martin Burg

	**Research topic**: Neurochemistry of *Drosophila*

	- Mapped mutations obtained through forward mutagenesis through a combination of fly genetics and molecular biology
	- Verified the neuronal expression pattern of histidine decarboxylase, the enzyme responsible for the synthesis of the neurotransmitter histamine, using a GFP reporter

Publications and presentations
---

\* **Denotes equal contribution**


**Preprints:**

<p class="p2"> Zeng Y, \*Zeng H, \*Fair BJ, Krishnamohan A, Hou Y, Hall JM, Ruthenburg AJ, Li YI, Staley JP. (2021). Profiling of Nascent Lariat Intermediates Reveals Key Genetic Determinants of the Timing of Human Co-transcriptional Splicing. *Bioarxiv*, 2021.10.18.464728. [doi: https://doi.org/10.1101/2021.10.18.464728](https://doi.org/10.1101/2021.10.18.464728). (In review at Mol Cell)</p> 


___


**Publications:**

<p class="p2"> Mu Z, Wei W, **Fair BJ**, Miao J, Zhu P, Li Y. (2021). The impact of cell type and context-dependent regulatory variants on human immune traits. *Genome Biology*, 22, 122. [PMID: 33926512](https://pubmed.ncbi.nlm.nih.gov/33926512/)</p>


<p class="p2"> **Fair BJ**, Blake LE, Sarkar A, Pavlovic BJ, Cuevas C, Gilad Y. (2020). Gene expression variability in human and chimpanzee populations share common determinants. *eLife*, 2020;9:e59929. [PMID: 33084571](https://pubmed.ncbi.nlm.nih.gov/33084571/) </p>


<p class="p2"> \* Xu H, \* **Fair BJ**, Dwyer Z, Gildea M, Pleiss JA. (2019). Detection of splice isoforms and rare intermediates using multiplexed primer extension sequencing. *Nature Methods*, 16, 55-58. [PMID: 30573814](https://pubmed.ncbi.nlm.nih.gov/30573814/) </p>


<p class="p2"> **Fair BJ**, Pleiss JA. (2017). The power of fission: yeast genetics as a tool for understand complex splicing. *Current Genetics* 63(3):375-380. doi: 10.1007/s00294-016-0647-6. [PMID:27628706](https://pubmed.ncbi.nlm.nih.gov/27628706/) </p>


<p class="p2"> \* Larson A, \* **Fair BJ**, & Pleiss JA. (2016). Interconnections Between RNA-Processing Pathways Revealed by a Sequencing-Based Genetic Screen for Pre-mRNA Splicing Mutants in Fission Yeast. *G3* (Bethesda, Md.), 6(6), 1513– 23. [PMID:27172183](https://pubmed.ncbi.nlm.nih.gov/27172183/) </p>


<p class="p2"> Vo TV, Das J, Meyer MJ, Cordero NA, Akturk N, Wei X, **Fair BJ** ... Yu H. (2016). A Proteome-wide Fission Yeast Interactome Reveals Network Evolution Principles from Yeasts to Human. *Cell*, 164(1-2), 310–323. [PMID:26771498](https://pubmed.ncbi.nlm.nih.gov/26771498/) </p>


----------


**Presentations:**


<p class="p2"> Fair BJ, Zeng Y, Mossain G, Lozano S, Staley J, Li YI,. “Natural genetic variation reveals mechanisms of chromatin state and splicing” (2022). Annual Meeting of the RNA Society. Boulder, CO. (poster) </p>


<p class="p2"> Fair BJ, Pleiss JA. “Identification of cis- and trans- regulatory elements necessary for cross-exon interactions during early spliceosome assembly in S. pombe.” (2016). EMBO Gene transcription in yeast. Sant Feliu de Guixol, Spain. (poster)</p>


<p class="p2"> Fair BJ, Larson A, Pleiss JA. “Genome-wide methods for identifying pre-mRNA splicing factors and measuring splicing in yeast.” (2016). American Cancer Society gala. Ithaca, NY. (poster)</p>


<p class="p2"> Fair BJ, Awan A, Pleiss JA. “Identifying Regulators of Alternative Splicing in the Fission Yeast Schizosaccharomyces pombe.” (2013). Finger Lakes RNA Conference. Canandaigua, NY. (oral presentation) </p>


<p class="p2"> Fair BJ, Awan A, Pleiss JA. “Identifying Regulators of Alternative Splicing in the Fission Yeast Schizosaccharomyces pombe.” (2013). Annual Meeting of the RNA Society. Quebec City, Quebec. (poster) </p>


<p class="p2">Fair BJ, VanderVilet M, Burg M. “Epitope labeling of histidine decarboxylase in D. melanogaster.” (2012). Genetics Society of America National Drosophila Conference. Chicago, IL. (poster) </p>


Teaching experience
----------------------------------------
2018
:	**Trained for lab technicians, graduate rotation students, and undergraduate students**
:	Trained three lab technicians, supervised an undergraduate lab researcher thesis project, and graduate rotation students towards executing both wet-lab molecular biology and computational genetics projects

2016
:	**Mentored rotation student projects**
:	Helped rotation students design and carry out NGS-based screens using automated pipetting robots in a fission yeast model system

2016
:	**Teaching Assistant in graduate level Advanced Biochemistry Laboratory course**
:	Helped students design, carry out, and analyze qRT-PCR and next-gen- sequencing experiments from tissue culture

2014, 2016
:	**Teaching Assistant in Introductory Cell and Developmental Biology course**
:	Led discussion sections, graded written work, contributed to exam-development

2010-2012
:	**Physics tutor**
:	Paid position at Grand Valley State University Tutoring Center. Tutored general physics classes.

Awards
---

2016
:	Cornell Center for Vertebrate Genomics Travel Award

2012
:	Cornell University Graduate Fellowship

2012
:	Most Outstanding Graduating Student Biology Major

Service and membership
---
2013, 2014
:	Expanding Your Horizons science workshop volunteer

2013
:	Cornell Dept. of Molecular Bio & Genetics grad-student recruitment
organizing committee

2012
:	Phi Kappa Phi Honor Society

References
---

**Yang I. Li, PhD**
:	**Relationship:** Postdoc advisor
:	**Title:** Associate Professor, Biological Sciences Division, University of Chicago
:	**Email:** yangili1@uchicago.edu

___

**Yoav Gilad, PhD**
:	**Relationship:** Postdoc advisor
:	**Title:** Professor, Biological Sciences Division, Chief of Section of Genetic Medicine, University of Chicago
:	**Email**: gilad@uchicago.edu

___

**Jeffrey A. Pleiss, PhD**
:	**Relationship:** PhD advisor
:	**Title:** Associate Professor, Department of Molecular Biology and Genetics, Cornell University
:	**Email:** jap332@cornell.edu
:	**Office phone:** (607) 254-8562
:	**Address:**
:	451 Biotechnology Building
:	Ithaca, NY 14853-2703

___

**Andrew Grimson, PhD**
:	**Relationship:** PhD thesis committee
:	**Title:** Associate Professor, Department of Molecular Biology and Genetics, Cornell University
:	**Email:** awg58@cornell.edu
:	**Office phone:** (607) 255-2453
:	**Address:**
:	445 Biotechnology Building
:	Ithaca, NY 14853-2703
